packages/project-config/src/paths.ts

```diff
@@ -153,7 +153,7 @@ export const getBaseDirFromFile = (file: string) => {
  */
 export const resolveFile = (
   filePath: string,
-  extensions: string[] = ['.js', '.tsx', '.ts', '.jsx'],
+  extensions: string[] = ['.js', '.tsx', '.ts', '.jsx', '.mjs', '.mts'],
```

Comment:
It is not, but I think we ought to support them, no? I was using it to test what happens when you change the vite config to vite.config.mts

Reply:
I'm not sure we should. We're not including any files like that in our templates, and we're not generating any files like that. Do you expect users to write .mjs and .mts files? Should we then also support .cjs and .cts?
I'm happy to be convinced either direction 🙂

Reply:
I only really wanted the ability to write a vite.config.mts - because right now the RW projects don't have type: module - to force Vite to use an ESM version of an import (lets say a plugin). Without this change, we just get "no vite.config found" - which is why I think mts and mjs should be allowed. 

I don't see a reason to use the cjs extensions!

Reply:
Sounds good to me!